packages/vanilla-renderers/src/controls/RadioGroup.tsx

```diff
@@ -70,10 +70,11 @@ export const RadioGroup = ({
   );
   const hasRadioClass = !radioControl || radioControl === 'radio';
   let groupStyle: { [x: string]: any } = {};
+  // compute flexDirection based on the optional option 'orientation=vertical|horizontal'
   if (hasRadioClass) {
     groupStyle = {
       display: 'flex',
-      flexDirection: 'row'
+      flexDirection: ('vertical' === uischema.options.orientation) ? 'column' : 'row'
```

Comment:
We need to access the `appliedUiSchemaOptions` instead of the `uischema.options` directly. Then probably the test cases will also work again.
```suggestion
      flexDirection: ('vertical' === appliedUiSchemaOptions.orientation) ? 'column' : 'row'
```